src/aiida_quantumespresso/workflows/xspectra/crystal.py

```diff
@@ -429,7 +443,48 @@ def validate_inputs(inputs, _):
                         'any wavefunction data.'
                     )
 
+        if 'symmetry_data' in inputs:
+            spacegroup_number = inputs['symmetry_data']['spacegroup_number'].value
+            equivalent_sites_data = inputs['symmetry_data']['equivalent_sites_data'].get_dict()
+            if spacegroup_number <= 0 or spacegroup_number >= 231:
+                raise ValidationError(
+                    f'Input spacegroup number ({spacegroup_number}) outside of valid range (1-230).'
+                )
```

Comment:
I notice that the rest of the validator also raises exceptions, but the proper usage of a port validator is to return a string in case of a problem. So all `raise ValidationError` should simply be replaced by `return`.